pkg/KubeArmorAnnotation/Makefile

```diff
@@ -95,6 +95,7 @@ delete: delete-cert-manager ## Delete controller from the K8s cluster specified
 deployment: manifests kustomize
 	cd config/manager; $(KUSTOMIZE) edit set image controller=${IMG}:latest
 	cd $(CURDIR); $(KUSTOMIZE) build config/default > ../../deployments/annotations/kubearmor-annotation-manager.yaml
+	sed -i 's/control-plane: controller-manager/kubearmor-app: annotation-manager/g' ../../deployments/annotations/kubearmor-annotation-manager.yaml
```

Comment:
remove this line and re-run 'make deployment' again in order to update ../../deployments/annotations/kubearmor-annotation-manager.yaml